SystemsGenetics/GEMmaker
# Ability to Skip Samples

Issue #111

## Description



This PR provides the ability to skip a sample.  This is useful in the event of a very large sample file.  If a sample continuously fails due to some factor that the user cannot control then they can add the sample to a "skip" file.  The next time GEMmaker is run the samples in the file will be skipped.  

This allows the user to not have to edit the original SRA_IDs.txt file to remove bad samples (which can get unwieldy for huge datasets).  

## Testing?

1. Pull the branch and run the example dataset.  
2.  Edit the file in `examples/samples2skip.txt` and add the sample ID: 1.  Run GEMmaker. This should skip the local sample with the label "1".
3.  Clean out the `work`, `output` directories.
3.  Next edit the file again and add the remote sample:  `SRX218012`.  Run GEMaker. Now both sample "1" and this remote sample should be missing.
